src/main/fc/settings.yaml

```diff
@@ -219,13 +219,13 @@ groups:
         default_value: 1000
         max: 9000
       - name: gyro_anti_aliasing_lpf_hz
-        description: "Gyro processing anti-aliasing filter cutoff frequency. In normal operation this filter setting should never be changed. In Hz"
-        default_value: 250
+        description: "Gyro processing anti-aliasing filter cutoff frequency. In normal operation this filter setting should never be changed. In samples"
+        default_value: 1
         field: gyro_anti_aliasing_lpf_hz
         max: 1000
       - name: gyro_main_lpf_hz
-        description: "Software based gyro main lowpass filter. Value is cutoff frequency (Hz)"
-        default_value: 60
+        description: "Software based gyro main lowpass filter. Value is samples"
+        default_value: 3
         field: gyro_main_lpf_hz
         min: 0
         max: 500
```

Comment:
LULU option should be added to `gyro_filter_mode` setting so we can have both LULU and classic approach available.
So `gyro_filter_mode` table shouyld have STATIC, DYNAMIC, ADAPTIVE and LULU options

Reply:
Implemented as requested on new push